model_servers/object_detection_python/tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+import pytest_container
+import pytest
+import os
+import logging
+import platform
+
+if 'PORT' not in os.environ:
+    PORT = 8000
+else:
+    PORT = os.environ['PORT']
+    try:
+        PORT = int(PORT)
+    except:
+        PORT = 8000
+
+if 'IMAGE' not in os.environ:
+    IMAGE = 'quay.io/ai-lab/model_servers/object_detection_python:latest'
```

Comment:
The container registry should be ghcr.io

Reply:
Dont images get tested against GHCR.io and promoted to quay? This will first check if `IMAGE` env var is present, and if not it would set the prod image. Happy to change it though if you can confirm this is incorrect.

Reply:
I am just going to assume your right haha we can always swap it back